coreneuron/permute/cellorder.cpp

```diff
@@ -601,13 +601,19 @@ void solve_interleaved2(int ith) {
 #endif
 
     int ncore = nwarp * warpsize;
+
+#ifdef ENABLE_CUDA_INTERFACE
+    NrnThread* d_nt = (NrnThread*) acc_deviceptr(nt);
+    InterleaveInfo* d_info = (InterleaveInfo*) acc_deviceptr(interleave_info + ith);
+    solve_interleaved2_launcher(d_nt, d_info, ncore);
+#else
```

Comment:
IMO, we should remove this `ENABLE_CUDA_INTERFACE` compile time switch and make the possibility to launch this via CLI option at runtime.